.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Report an issue or bug with the Quantum Core X.
+title: "bug: "
+labels: [ "type/bug" ]
```

Comment:
Ah, sorry, I forgot this one - the labels did not appear to work for me:

![image](https://github.com/user-attachments/assets/c20fee8a-2916-436a-bbfc-75401a66e1fe)

Do these have to exist beforehand?

![image](https://github.com/user-attachments/assets/2450c72a-4aa7-468d-a10a-fbd2070514de)

Reply:
I'm not entirely sure, but I think those labels are the default ones from GitHub. Since we're using custom labels, it's normal that they don't show up.

Reply:
I'm certain these are the default labels. This can be ignored I would say